### DISCUSSION TITLE
Config issue in dockerfile

### DISCUSSION THREAD

**Discussion by SerdarNur** [verdaccio/verdaccio#3731] [Q&A]
Hello together,

i tried today this example https://github.com/verdaccio/verdaccio/tree/master/docker-examples/v6/docker-local-storage-volume and everything worked fine. After i tried to use the command npm adduser i received the following error:

![image](https://user-images.githubusercontent.com/7069847/231430897-a2c6d834-f489-4917-93ee-1f1198caf4d6.png)

I was confused because my volumes were correct. Else it would not start with success.

![image](https://user-images.githubusercontent.com/7069847/231431263-3bbd3773-15c4-4632-9ec1-7af9577b7b50.png)

The workaround was that i needed to create in the conf folder again a folder named verdaccio/conf with the same content. Then it worked.

Is there an issue or misconfiguration by my side?

**Comment by juanpicado** [comment]
I think you need to add higher permissions to the volumes folder yourself

**Comment by SerdarNur** [comment]
@juanpicado  no it was not about permissions, because then the docker container would not start.

**Comment by pchong-msrl** [comment]
I have face the same issue, please fix that

**Reply by juanpicado** [reply]
Or you could :) is an example so feel free to contribute

**Comment by pchong-msrl** [comment]
for anyone who have face the same issue. You need to chown to the mounted folder before docker-compose up
https://verdaccio.org/docs/docker#:~:text=sudo%20chown%20%2DR%2010001%3A65533%20/path/for/verdaccio
